html-extract.js

```diff
@@ -41,7 +42,7 @@ module.exports = function (opts) {
           self.push(new gutil.File({
             // Name: id or tag + index.
             path: file.path + "-" + (el.attribs.id || el.tagName + "-" + i),
-            contents: new Buffer(el.children[0].data)
+            contents: new Buffer(trim.left(el.children[0].data))
```

Comment:
oh, I totally missed the `trim.left()` thing.

I actually looked at the source: https://github.com/component/trim/blob/master/index.js#L11 and the whole lib is three "real" lines of code.

How about "proposal (revised)":

```js
(el.children[0].data || "").replace(/^\s*/, ""); // trim left
```

?